code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -0,0 +1,400 @@
+		target = H
+		mode = BOT_HUNT
+
+/mob/living/simple_animal/bot/honkbot/attack_hand(mob/living/carbon/human/H)
+	if(H.a_intent == "harm")
+		retaliate(H)
+		spawn(5)
+			playsound(loc, 'sound/machines/buzz-sigh.ogg', 50, 1, -1)
+	return ..()
+
+
+/mob/living/simple_animal/bot/honkbot/attackby(obj/item/W, mob/user, params)
+	..()
+	if(istype(W, /obj/item/weldingtool) && user.a_intent != INTENT_HARM).
+		return
+	if(!istype(W, /obj/item/screwdriver) && (W.force) && (!target) && (W.damtype != STAMINA) ) // Check for welding tool to fix #2432.
+		retaliate(user)
+		spawn(5)
+			playsound(loc, 'sound/machines/buzz-sigh.ogg', 50, 1, -1)
+
+/mob/living/simple_animal/bot/honkbot/emag_act(mob/user)
+	..()
+	if(emagged == 2)
+		if(user)
+			user << "<span class='danger'>You short out [src]'s sound control system.</span>"
+			oldtarget_name = user.name
+		audible_message("<span class='danger'>[src] buzzes oddly!</span>")
+		playsound(loc, 'sound/machines/buzz-sigh.ogg', 50, 1, -1)
+		icon_state = "honkbot[on]"
+
+/mob/living/simple_animal/bot/honkbot/bullet_act(obj/item/projectile/Proj)
+	if(istype(Proj ,/obj/item/projectile/beam)||istype(Proj,/obj/item/projectile/bullet))
+		if((Proj.damage_type == BURN) || (Proj.damage_type == BRUTE))
+			if(!Proj.nodamage && Proj.damage < src.health)
+				retaliate(Proj.firer)
+	..()
+
+/mob/living/simple_animal/bot/honkbot/UnarmedAttack(atom/A)
+	if(!on)
+		return
+	if(iscarbon(A))
+		var/mob/living/carbon/C = A
+		if (!emagged)
+			honk_attack(A)
+		else
+			if(!C.IsStun() || arrest_type)
+				stun_attack(A)
+		..()
+	else if (!spam_flag) //honking at the ground
+		bike_horn(A)
+
+
+/mob/living/simple_animal/bot/honkbot/hitby(atom/movable/AM, skipcatch = 0, hitpush = 1, blocked = 0)
+	if(istype(AM, /obj/item))
+		playsound(loc, honksound, 50, 1, -1)
+		var/obj/item/I = AM
+		if(I.throwforce < src.health && I.thrownby && (istype(I.thrownby, /mob/living/carbon/human)))
+			var/mob/living/carbon/human/H = I.thrownby
+			retaliate(H)
+	..()
+
+/mob/living/simple_animal/bot/honkbot/proc/bike_horn() //use bike_horn
+	if (!emagged)
+		if(ckey == null) //check if a player is controlling
+			playsound(loc, honksound, 50, 1, -1)
+		else
+			if (!spam_flag)
+				playsound(loc, honksound, 50, 1, -1)
+				spam_flag = 1 //prevent spam
+		icon_state = "honkbot-c"
+		spawn(5)
+			icon_state = "honkbot[on]"
+		spawn(cooldowntimehorn)
+			spam_flag = 0
+
+	else //emagged honkbots will spam short and memorable sounds.
+
+		if (ckey == null)
+			playsound(loc, pick('sound/items/bikehorn.ogg', 'sound/items/AirHorn2.ogg', 'sound/misc/sadtrombone.ogg', 'sound/items/AirHorn.ogg', 'sound/effects/reee.ogg', 'sound/effects/adminhelp.ogg', 'sound/items/WEEOO1.ogg', 'sound/voice/biamthelaw.ogg', 'sound/voice/bcreep.ogg','sound/magic/Fireball.ogg' ,'sound/effects/pray.ogg', 'sound/voice/hiss1.ogg','sound/machines/buzz-sigh.ogg', 'sound/machines/ping.ogg', 'sound/weapons/flashbang.ogg', 'sound/weapons/bladeslice.ogg'), 50, 0)
```

Comment:
Take out the adminhelp.ogg, we do NOT want to train players to ignore that sound.

Reply:
At 50% volume it was easily distinguishable from the actual admin bwoink.

Removing it now.